### ISSUE TITLE
Month parameter ignored by Power Network Region By Fueltech API call

### ISSUE THREAD

**Issue by ekemini-e** [opennem/opennem#393]
The api Power Network Region By Fueltech query is supposed to accept a "month" parameter to enable historical data to be accessed see [here](https://api.opennem.org.au/docs#tag/Stats/operation/Power_Network_Region_by_Fueltech_stats_power_network_fueltech__network_code___network_region_code__get) This is however being ignored, and last 7 day data is being returned instead. 

I believe I have identified why this is happening and I have created [PR # 392](https://github.com/opennem/opennem/pull/392) to address this. @nc9 Can you please review and merge this PR?

Thanks

**Comment by nc9** [comment]
fixed in #626, merged to main.

@ekemini-e your diagnosis in #392 was correct and the fix is essentially yours — crediting it in the commit. sorry it sat this long.

the mechanism, for the record: `month` is defaulted to today at the top of the handler, before the static-redirect check runs.

```python
if not month:
    month = get_today_nem().date()
...
if settings.redirect_api_static:
    return RedirectResponse(url=".../power/7d.json")
```

so by the time the redirect is evaluated, every request looks like a current one — the redirect always fires and you get the last 7 days from `7d.json` no matter which month you asked for. `redirect_api_static` is on in prod, which is why it reproduced for everyone.

now it captures whether a month was actually requested before defaulting, and only redirects when it wasn't. also checked the rest of the codebase — this is the only use of `redirect_api_static`, so no sibling endpoint has the same shape.

worth being straight about the status of this endpoint: it's deprecated, hidden from the schema, and superseded by the v4 api on the platform. it's fixed because it's still mounted and still serving wrong data, not because v3 is coming back. for anything new, v4 is the one to build against.

@VIKTORVAV99 this should unblock the historical data you were after.
